ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -17,7 +17,7 @@ android {
     compileSdkVersion 30
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 30
+        targetSdkVersion 29
```

Comment:
this should be 30

Reply:
that's a curious error, annotations did show it was done on purpose. anyway reverting